boofuzz/primitives/fuzz_list.py

```diff
@@ -0,0 +1,125 @@
+import random
+import glob
+
+from .base_primitive import BasePrimitive
+
+
+class FuzzList(BasePrimitive):
```

Comment:
This could be simplified by utilizing more of what `BasePrimitive` gives us for free. `delim.py` is an excellent example of a primitive that uses `BasePrimitive` well. Basically you can just add stuff to `_fuzz_library` and `BasePrimitive` does the rest of the work.